vertical-pod-autoscaler/pkg/updater/main.go

```diff
@@ -51,12 +50,12 @@ import (
 )
 
 var (
-	updaterInterval = flag.Duration("updater-interval", 1*time.Minute,
-		`How often updater should run`)
-
 	minReplicas = flag.Int("min-replicas", 2,
 		`Minimum number of replicas to perform update`)
 
+	updaterInterval = flag.Duration("updater-interval", 1*time.Minute,
+		`How often updater should run`)
+
```

Comment:
Super nitpick, sorry about that.
Can this be put back where it was?
Just to keep git history clean.

Reply:
yes, of course

Reply:
Done